packages/server/src/healthcheck.ts

```diff
@@ -73,6 +74,11 @@ export async function healthcheckHandler(_req: Request, res: Response): Promise<
     BASE_METRIC_OPTIONS
   );
 
+  const subscriptionQueue = getSubscriptionQueue();
+  if (subscriptionQueue) {
+    setGauge('medplum.bullmq.subscriptionQueueLength', await subscriptionQueue.getWaitingCount());
```

Comment:
I'll bikeshed a more general/descriptive name for the metric, e.g`medplum.subscriptions.enqueuedCount` or `medplum.queue.subscription.enqueuedCount`

Reply:
Went with `medplum.subscription.waitingCount` since we have `medplum.subscription` namespace already and `waiting` is the name of the status, which only includes jobs currently waiting to be picked up and doesn't include `delayed` jobs.

Actually now that I'm thinking about it should we have both of those metrics?

Reply:
@ThatOneBro 👍 to both!